package/boot/grub2/Makefile

```diff
@@ -51,44 +51,174 @@ define Package/grub2-editenv/description
 	Edit grub2 environment files.
 endef
 
+ifneq ($(CONFIG_GRUB_EFI),)
+GRUB_FONT:=unifont
+GRUB_FONT_VERSION:=9.0.06
+GRUB_FONT_SOURCE:=$(GRUB_FONT)-$(GRUB_FONT_VERSION).bdf.gz
+GRUB_FONT_SOURCE_URL:=@GNU/unifont
+GRUB_FONT_SOURCE_HASH:=4246c4773ed70f78a7e27ff1118fd257a280d1102200265ad5d58bb2011195ef
+
+define Download/grub_font
+  FILE:=$(GRUB_FONT_SOURCE)
+  URL:=$(GRUB_FONT_SOURCE_URL)
+  HASH:=$(GRUB_FONT_SOURCE_HASH)
+endef
+endif
+
 HOST_BUILD_PREFIX := $(STAGING_DIR_HOST)
 
-CONFIGURE_VARS += \
+HOST_CONFIGURE_CMD := $(BASH) ../configure
+
+BIOS_CONFIGURE_VARS += \
 	grub_build_mkfont_excuse="don't want fonts"
 
-CONFIGURE_ARGS += \
+BIOS_CONFIGURE_ARGS += \
```

Comment:
Shouldn't this be `HOST_BIOS_CONFIGURE_ARGS` (like below)?  I'm not seeing `BIOS_CONFIGURE_ARGS` being used anywhere...